docs/user-guide/profile.md

```diff
@@ -37,15 +37,20 @@ When you're in Profle page, you can see your identity list
 
 ![image](https://github.com/linagora/tmail-flutter/assets/68209176/9a02fdae-6aa6-46c2-b599-1bc4d54fb717)
 
+- You can include an image in signature of the identity, .eg company logo, product logo, project logo.... The image is sent in every email, so it should be an small image (less than 16KB).
+
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/f114e9c6-5674-42cb-9fc7-6d7404bc972f)
+
+
```

Comment:
Please add a section on selecting the default identity.

Reply:
It is described above :
![image](https://github.com/linagora/tmail-flutter/assets/68209176/79100bc3-4341-4f67-bb0e-dee35e1ca3f2)